packages/main/src/themes/sap_horizon_hcb/Switch-parameters.css

```diff
@@ -17,34 +14,23 @@
 
 	--_ui5_switch_root_outline_top: 0.5rem;
 	--_ui5_switch_root_outline_bottom: 0.5rem;
-	--_ui5_switch_root_outline_left: 0.375rem;
-	--_ui5_switch_root_outline_right: 0.375rem;
-
-	--_ui5_switch_root_compact_outline_top: 0.25rem;
-	--_ui5_switch_root_compact_outline_bottom: 0.25rem;
+	--_ui5_switch_root_outline_left: -0.125rem;
+	--_ui5_switch_root_outline_right: -0.125rem;
 
 	--_ui5_switch_disabled_opacity: var(--sapContent_DisabledOpacity);
 
 	--_ui5_switch_transform: translateX(100%) translateX(-1.625rem);
-	--_ui5_switch_compact_transform: translateX(100%) translateX(-1.375rem);
 	--_ui5_switch_transform_with_label: translateX(100%) translateX(-1.875rem);
-	--_ui5_switch_compact_transform_with_label: translateX(100%) translateX(-1.875rem);
 
 	--_ui5_switch_rtl_transform: translateX(-100%) translateX(1.625rem);
-	--_ui5_switch_compact_rtl_transform: translateX(1.375rem) translateX(-100%);
 	--_ui5_switch_rtl_transform_with_label: translateX(-100%) translateX(1.875rem);
-	--_ui5_switch_compact_rtl_transform_with_label: translateX(1.875rem) translateX(-100%);
```

Comment:
Why do we remove more in this file? I see that they are compact related?

Reply:
Those are forgotten variables, that are not used anywhere, so I just removed them. They are not related to the change specifically, but rather a clean-up.